Library/Homebrew/test/global_spec.rb

```diff
@@ -7,9 +7,4 @@
       .and not_to_output.to_stderr
       .and be_a_success
   end
-
-  it "does not require i18n" do
-    # This is a transitive dependency of activesupport, but we don't use it.
-    expect { I18n }.to raise_error(NameError)
```

Comment:
This no longer passes typechecking, bc the i18n rbi file is gone. I don't think it's an especially useful test anymore, so let's just yank it.